plugins/src/hashlookup/select.go

```diff
@@ -0,0 +1,241 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/blastrain/vitess-sqlparser/sqlparser"
+)
+
+// If the where is empty, need to check whether to agg or not
+func checkNeedAgg(sqlSelect sqlparser.SelectExprs) bool {
+	for _, v := range sqlSelect {
+		expr, ok := v.(*sqlparser.AliasedExpr)
+		if !ok {
+			// No need to handle, star expression * just skip is ok
+			continue
+		}
+
+		// TODO: more precise
+		if _, ok := expr.Expr.(*sqlparser.FuncExpr); ok {
+			return true
+		}
+	}
+
+	return false
+}
+
+func handleSelectWhereComparisonExpr(expr *sqlparser.Expr, topLevel bool, parent *sqlparser.Expr) ([][2]string, error) {
```

Comment:
/*
 * Handle single "field operator value" expression.
 *
 * Receives:
 * expr     - SQL expression to process
 * topLevel - whether it's a top level expression
 * parent   - container of the expression
 */